exporters/elasticsearch/src/es_log_recordable.cc

```diff
@@ -9,6 +11,27 @@
 #include "opentelemetry/trace/trace_flags.h"
 #include "opentelemetry/trace/trace_id.h"
 
+namespace nlohmann
+{
+template <>
+struct adl_serializer<opentelemetry::sdk::common::OwnedAttributeValue>
+{
+  static void to_json(json &j, const opentelemetry::sdk::common::OwnedAttributeValue &v)
+  {
+    opentelemetry::nostd::visit([&j](const auto &value) { j = value; }, v);
```

Comment:
A possible breaking change here is that the original `WriteValue()` did not handle spans. If this is important, I can add an `if` condition to follow the old behavior.

Reply:
I just bumped into your changes and I was wondering if this could help putting json objects within the root json ?
The modifications I did was kind of a workaround so we could add keys like `user.id` that would be visible within ELK like the documentation asks for but the key was still `user.id` and not an object `user` under `root` that would have a key `id`

My hope would be to be able to be able to create an attribute or resource that contains a json object for some keys:

```
auto attr = { "user":  R"({"id": 1234}"_json};
```